tests/garage/tf/core/test_networks.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+
+import tensorflow as tf
+from garage.tf.core.networks import MLPs
+import numpy as np
+
+
+class TestNetworks(unittest.TestCase):
+    def test_networks(self):
```

Comment:
You may want to take a look at the test suite for lasagne to get some ideas for tests you can run on NN layers.

https://github.com/Lasagne/Lasagne